CHANGELOG.md

```diff
@@ -1,18 +1,56 @@
 Changes by Version
 ==================
 
-1.15.0 (unreleased)
+1.15.0 (2019-11-07)
 ------------------
-
 #### Backend Changes
 
 ##### Breaking Changes
 
-* The default value for the Ingester's flag `ingester.deadlockInterval` has been changed to `0`. With the new default, the ingester won't `panic` if there are no messages for the last minute. To restore the previous behavior, set the flag's value to `1m`.
+* The default value for the Ingester's flag `ingester.deadlockInterval` has been changed to `0` ([#1868](https://github.com/jaegertracing/jaeger/pull/1868), [@jpkrohling](https://github.com/jpkrohling))
+
+  With the new default, the ingester won't `panic` if there are no messages for the last minute. To restore the previous behavior, set the flag's value to `1m`.
 
+* CLI flags have below changes: 
```

Comment:
-1 on documenting added flags.

If some flags changed please list old deprecated flags and mention if they are working or they were removed.
Could you please add a link to PR which changed the flags.